app/views/assays/show.html.erb

```diff
@@ -124,7 +134,7 @@
   <%= tab_pane('related-items') do %>
     <%= render partial: 'general/items_related_to', object: @assay %>
   <% end %>
-	<% if Seek::Config.isa_json_compliance_enabled && displaying_single_page? %>
+	<% if Seek::Config.isa_json_compliance_enabled && @assay.is_ISA_JSON_compliant %>
```

Comment:
I don't know if you were planning on fixing it at a later stage but I just want to mention here that this will not work without modifying the dynamic table logic.
It requires a selected 'Samples table', which represents a sample type, from the treeview.

Reply:
Yes, we are aware that this requires future modification of the dynamic table logic to work (and that it hence currently does not).